lib/deployments/core.js

```diff
@@ -22,19 +26,21 @@ const createDeployment = async function(args) {
         ['VaultManagerParameters', 'VaultParameters'],
         ['VaultManagerBorrowFeeParameters', 'VaultParameters', baseBorrowFeePercent, borrowFeeReceiver],
         ['CDPManager01', 'VaultManagerParameters', 'OracleRegistry', 'CDPRegistry', 'VaultManagerBorrowFeeParameters'],
-        ['ForceTransferAssetStore', 'VaultParameters', []],
-        ['LiquidationAuction02', 'VaultManagerParameters', 'CDPRegistry', 'ForceTransferAssetStore'],
+        ['CDPManager01_Fallback', 'VaultManagerParameters', 'OracleRegistry', 'CDPRegistry', 'VaultManagerBorrowFeeParameters'],
+        ['AssetsBooleanParameters', 'VaultParameters', ['0x4bfB2FA13097E5312B19585042FdbF3562dC8676', '0x988AAf8B36173Af7Ad3FEB36EfEc0988Fbd06d07'], [PARAM_FORCE_TRANSFER_ASSET_TO_OWNER_ON_LIQUIDATION, PARAM_FORCE_TRANSFER_ASSET_TO_OWNER_ON_LIQUIDATION]],
```

Comment:
I hardcoded adresses since it is the branch for ethereum and to not to forget to add these assets to contract. Supposed that in branches for oher networks/dev will be stored only addresses for current network.

But yes, now I also think that we shouldn't do this. Will remove.

Reply:
removed